src/SFA.DAS.AANHub.Database/Tables/JobAudit.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[JobAudit]
+(
+    [Id] INT NOT NULL IDENTITY(1, 1),
+	[JobName] NVARCHAR(250) NOT NULL , 
+    [StartTime] DATETIME2 NOT NULL, 
+    [EndTime] DATETIME2 NOT NULL,
+    [Notes] NVARCHAR(MAX) NULL,
+    CONSTRAINT [PK_JobAudit] PRIMARY KEY ([Id])
```

Comment:
We probably need to include an index here as we will be retrieving the data by JobName and StartTime. 
`INDEX [IX_JobAudit_JobNameStartTime] NONCLUSTERED ([JobName], [StartTime])